client/web/compose/src/components/ModuleFields/Configurator/validation.vue

```diff
@@ -158,6 +235,37 @@ export default {
       const [year, month] = VERSION.split('.')
       return `https://docs.cortezaproject.org/corteza-docs/${year}.${month}/integrator-guide/compose-configuration/index.html`
     },
+    modifierOptions () {
+      return [
+        { value: 'ignore-case', text: this.$t('constraints.ignoreCase') },
+        { value: 'fuzzy-match', text: this.$t('constraints.fuzzyMatch') },
+        { value: 'sounds-link', text: this.$t('constraints.soundsLike') },
+      ]
+    },
+    multiValueOptions () {
+      return [
+        { value: 'one-of', text: this.$t('constraints.oneOf') },
+        { value: 'equal', text: this.$t('constraints.equal') },
+      ]
+    },
+    constraint: {
+      get () {
+        if (this.module.config.recordDeDup.rules[this.ruleIndex]) {
+          return this.module.config.recordDeDup.rules[this.ruleIndex].constraints[this.constraintIndex]
+        }
+
+        return {}
+      },
+      set (value) {
+        if (this.module.config.recordDeDup.rules[this.ruleIndex]) {
+          this.module.config.recordDeDup.rules[this.ruleIndex].constraints[this.constraintIndex] = value
+        }
+      },
+    },
+  },
+
+  created () {
```

Comment:
do mounted just in case if it remains created in the background (probably doesn't but you never know with modals)